snc.sh

```diff
@@ -253,6 +250,12 @@ retry ${OC} delete mc chronyd-mask
 # Wait for the cluster again to become stable because of all the patches/changes
 wait_till_cluster_stable
 
+# Replace pull secret with a null json string '{}'
+retry ${OC} replace -f pull-secret.yaml
```

Comment:
Will this help with crc downloading a lot of  data right after initial start? Or is this unrelated?

Reply:
@cfergeau it is unrelated, downloading the data still happen if any image on marketplace is updated.